text/0032-nft-standard.md

```diff
@@ -69,14 +69,43 @@ The metadata should have the following JSON structure:
         "type": "string",
         "description": "A URL pointing to an immutable resource with a digital asset to which this token represents"
     },
+    "collection": {
+        "type": Object{CollectionObject},
+        "description": "Additional details are recommended for NFT collections with multiple unique assets under one single family and symbol."
+    }
     "attributes": {
         "type": "Array<AttributeObject>",
         "description": "These are the extra attributes for the digital asset. It's an array of AttributeObject to make it as flexible as possible."
     },
+    "royalty": {
+        "type": "Object<RoyaltyObject>",
+        "description": "Required fields that would be utilised to pay royalty to original creators in secondary sales conducted by the exchange that adheres Hathor Metadata standards, after minting (primary sale)."
+    },
+    "external_info": {
+        "type": "Object{InfoObject}",
+        "description": "Additional information for collection, to be used by exchange or custom NFT assets, e.g. Gaming Assets"
+    }
+}
+```
+
+- The CollectionObject has the following JSON structure:
+
+```
```

Comment:
To keep the same standard as in the `AttributeObject`, we must have `{` to start the object and `}` to close it. Similar to what you did in the `CreatorObject`.